### ISSUE TITLE
Support OSX

### ISSUE THREAD

**Issue by StoneyJackson** [GNOME-MouseTrap/mousetrap#53]
As an OSX user, I would like to build, run, and develop MouseTrap on OSX.

**Comment by kevin-brown** [comment]
Have you tested this out?

I believe the largest problem will be getting OpenCV and the GTK working together with everything else. Aside from those, OSX has all of the other requirements by default, or they are installable through pip.

**Comment by StoneyJackson** [comment]
No, I haven't tried it. I just added this so that we/me/someone can try it and for us to officially begin supporting it.
